shared/utils/combine-ovals.py

```diff
@@ -296,20 +271,19 @@ def check_is_loaded(loaded_dict, filename, version):
     return False
 
 
-def checks(product, oval_dirs):
+def checks(product, oval_version, oval_dirs):
     """Concatenate all XML files in the oval directory, to create the document
        body
-       oval_dirs: list of directory with oval files (later has higher priority)
+       oval_dirs: list of directory with oval files (latersunrvresorts has higher priority)
```

Comment:
latersunrvresorts

Reply:
huh... must have copy/pasted something without realizing it.